Annotation/constituent_golden_set.json

```diff
@@ -11,24 +11,7 @@
                     }
                 ],
                 "type": "constituent",
-                "clause": "",
-                "embedded clauses": []
-            }
-        ]
-    },
-    "27": {
-        "sentence": "And the experience is still paying dividends; this is where I learned to storyboard as well as to improvise when the storyboard became incompatible with reality.",
-        "embedded clauses": [
-            {
-                "predicate": [
-                    {
-                        "str": "improvise",
-                        "lemma": "",
-                        "POS": ""
-                    }
-                ],
-                "type": "constituent",
-                "clause": "",
+                "clause": "how to bring our video archives into this new Internets thingy",

```

Comment:
This is the clause from the first sentence in the set. Also, it seems like this might not be an interrogative embedded clause but a temporal relative clause

Reply:
so the stuff highlighted in red means I've deleted it, so I've removed the entry you have a problem with (because agree) and the clause you've highlighted is a part of the above entry